### ISSUE TITLE
[Bug]: `fzf.vim`, if invoked on startup, causes `lunarvim` to quit if closed

### ISSUE THREAD

**Issue by sumit0190** [LunarVim/LunarVim#1738]
### Problem description

1. Install the `fzf` and `fzf.vim` plugins.
2. Start `lunarvim`, and use the `:Files` command to open `fzf`.
3. Now hit `esc` (or whatever you have `esc` binded to).
4. Notice that instead of quitting the `fzf` window, `lunarvim` itself closes.
5. Now restart `lunarvim`, and this time, first open a file.
6. Open `fzf` again using `:Files`, and then hit `esc`. This time it closes as expected.

Somehow it only ever happens when doing it the first time, without anything loaded in the buffer. I am not sure if that's expected functionality or a bug, so feel free to close if necessary.

### LunarVim version

rolling

### Neovim version (>= 0.5)

0.5.1

### Relevant log output

```shell
NVIM v0.5.1
Build type: Release
LuaJIT 2.1.0-beta3
Compilation: clang -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -O2 -DNDEBUG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/tmp/neovim-20210927-1441-demfsz/neovim-0.5.1/build/config -I/tmp/neovim-20210927-1441-demfsz/neovim-0.5.1/src -I/usr/local/include -I/Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk/usr/include -I/usr/local/opt/gettext/include -I/tmp/neovim-20210927-1441-demfsz/neovim-0.5.1/build/src/nvim/auto -I/tmp/neovim-20210927-1441-demfsz/neovim-0.5.1/build/include
Compiled by brew@BigSur

Features: +acl +iconv +tui
```

### I have

- [X] Read the readme
- [X] Read the wiki
- [X] Searched for similar issues

**Comment by sumit0190** [comment]
Actually, this is also reproducible by just opening the main actions menu (by pressing the leader key), and then immediately closing it by pressing `esc`.

**Comment by sumit0190** [comment]
This has nothing to do with `fzf`, as I mentioned in my second comment. Just opening lunarvim in a folder, then pressing the `leader` key followed by `esc` does the same thing - it quits lunarvim, instead of just quitting the actions menu.

**Comment by kylo252** [comment]
Can you see if the issue persists with alpha.nvim? #1906

**Comment by waltermaldonado** [comment]
The same happens if you open a folder with `lvim .` and then start telescope with `:Telescope`. At the moment you hit `Esc` to close telescope window, LunarVim goes down as well.

**Comment by sumit0190** [comment]
@waltermaldonado You are right, that's a much simpler repro step. To anyone else who might be reading this, note that this **does not happen** if `lvim` is launched without opening a folder (so just running `lvim` or `lvim <filename>` does not cause this issue).

**Comment by kylo252** [comment]
Works correctly since the switch to alpha.nvim.
